app/models/group_assignment.rb

```diff
@@ -0,0 +1,6 @@
+class GroupAssignment < ApplicationRecord
+  self.table_name = :groups_users
```

Comment:
This seems like it will be really annoying later on. Any reason we have decided on this approach?